dist/profile/manifests/buildagent.pp

```diff
@@ -67,6 +73,60 @@
         'zip',
     ])
 
+    # There is no linux_aarch64 azcopy release, considering that aarch64 = amd64 so vagrant can run on Mac Silicon
+    $architecture = $facts['os']['architecture'] ? {
+      'aarch64' => 'arm64',
+      default   => $facts['os']['architecture'],
+    }
+    $azcopy_url = "https://azcopyvnext.azureedge.net/releases/release-10.21.0-20230928/azcopy_linux_${architecture}_10.21.0.tar.gz"
+
+    exec { 'Install azcopy':
+      require => [Package['curl'], Package['tar'], Account[$user]],
```

Comment:
```suggestion
      require => [Package['curl'], Package['tar']],
```

Didn't realize earlier but the account creation is not required for this to be executed